### PR TITLE
samba: send extra information

### PR DESCRIPTION
**Pull request by b-ranto**
Signed-off-by: Boris Ranto <branto@redhat.com>

### CODE REVIEW

File: samba/build/build_deb [hunk 1]
```diff
@@ -50,6 +50,21 @@ chacra_endpoint="samba/${BRANCH}/${GIT_COMMIT}/${DISTRO}/${DIST}"
 
 # push binaries to chacra
 find *.deb | $VENV/chacractl binary ${chacra_flags} create ${chacra_endpoint}/${DEB_ARCH}/
+    PACKAGE_MANAGER_VERSION=$(dpkg-deb -f $(find *"$DEB_ARCH".deb | head -1) Version)
```
Comment (alfredodeza) [inline samba/build/build_deb]: why is this indented?

File: samba/build/build_rpm [hunk 2]
```diff
@@ -50,6 +50,21 @@ chacra_endpoint="samba/${BRANCH}/${GIT_COMMIT}/${DISTRO}/${RELEASE}"
 
 # push binaries to chacra
 find *.rpm | $VENV/chacractl binary ${chacra_flags} create ${chacra_endpoint}/$ARCH/
+    PACKAGE_MANAGER_VERSION=$(rpm --queryformat '%{VERSION}-%{RELEASE}\n' -qp $(find *.rpm | egrep "\.$ARCH\.rpm" | head -1))
```
Comment (alfredodeza) [inline samba/build/build_rpm]: same here

**Comment by b-ranto** [comment]
Ah, copypasta mistake, should be fixed, now.

**Comment by alfredodeza** [comment]
once this gets rebased it is good to go

**Comment by b-ranto** [comment]
rebased

**Pull request merged**